start-valheim-server.sh

```diff
@@ -39,7 +39,15 @@ echo "Valheim port is: $VALHEIM_PORT"
 echo "Valheim server name is: $VALHEIM_SERVER_NAME"
 echo "Valheim world name is: $VALHEIM_WORLD_NAME"
 
+steamcmd/steamcmd.sh +login anonymous \
++force_install_dir $VALHEIM_SERVER_DIR \
++app_update 896660 \
+validate +exit > "/home/steam/valheim-data/steamcmd_log.txt"
+
```

Comment:
Does this mean every time the container is started, `steacmd` is used to to update the Valheim server?

Reply:
Yes, this is also how other steamcmd based docker images work.
It is also working like you would expect: if there is an update, it will download that update (like the desktop client only a delta update). If there is no update, it will just do no update